tracing/logger.go

```diff
@@ -196,6 +195,14 @@ func (logger *TraceLogger) AddEnvironment() error {
 		}
 		logger.NoSingleSpans = tcss == "true"
 	}
+	tdb := strings.ToLower(os.Getenv("TRACING_DEBUG"))
```

Comment:
A small nit but perhaps add `TRACING_DEBUG` to the method comment?

Reply:
Documentation is more than a nitpick!

Reply:
Also added to README.md